src/network/types.rs

```diff
@@ -73,7 +74,7 @@ pub struct PerNetworkOptions {
     pub interface_name: String,
 
     #[serde(rename = "static_ips")]
-    pub static_ips: Option<Vec<String>>,
+    pub static_ips: Option<Vec<IpAddr>>,
```

Comment:
thanks for switching this. I wanted to switch but was not sure if some other part of code is using this as string.